<launch>

  <node pkg="sbpl_arm_planner_node" type="sbpl_arm_planner_node" name="right_arm/sbpl_planner" output="screen" respawn="false">
  <!--   <node launch-prefix="gdb" pkg="sbpl_arm_planner_node" type="sbpl_arm_planner_node" name="right_arm/sbpl_planner" output="screen" respawn="false"/> -->

    <param name="env_config" textfile="$(find sbpl_arm_planner)/env_examples/robarm3d/r_arm.cfg" />
    <param name="planner_config" textfile="$(find sbpl_arm_planner)/params.cfg" />

    <param name="allocated_time" value="40.0" />
    <param name="forward_search" value="true" />
    <param name="search_mode" value="true" />
    <param name="num_joints" value="7" />
    <param name="planner_type" value="cartesian" />
    <param name="use_dijkstra_heuristic" value="true" />

    <param name="torso_arm_offset_x" value="0.0" />
    <param name="torso_arm_offset_y" value="-0.188" />
    <param name="torso_arm_offset_z" value="0.0" />

    <param name="collision_map_topic" value="collision_map_occ" />
    <param name="arm_name" value="right_arm" />
    <param name="lowres_cc" value="true" />
    <param name="use_collision_map" value="true" />
    <param name="use_collision_map_occ_to_update_voxel" value="true" />
    <param name="use_exact_gripper_collision_checking" value="false" />
    <param name="use_voxel3d_grid" value="true" />
    <param name="seconds_per_waypoint" value="0.240" />
    <param name="padding" value="0.03" />
    <param name="num_smoothing_iterations" value="5" />
    
    <!-- planning monitor -->
    <remap from="robot_description" to="robot_description" />
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />

    <param name="arm" type="string" value="right_arm" />
    <param name="planning_frame_id" type="string" value="torso_lift_link" />
    <param name="refresh_interval_collision_map" type="double" value="5.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />

  </node>

</launch>
